### PR TITLE
Added FC annual summary to data specs

### PR DESCRIPTION
**Pull request by fangfy**

### CODE REVIEW

File: data_specs/Fractional_Cover_specs.ipynb [hunk 1]
```diff
@@ -70,7 +77,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Coverage and metadata of Fractional Cover can be viewed on DE Africa [Metadata Explorer](https://explorer.digitalearth.africa/products/fc_ls)."
+    "Coverage and metadata can be viewed on DE Africa [Metadata Explorer](https://explorer.digitalearth.africa/products) for:\n",
```
Comment (alexgleith) [inline data_specs/Fractional_Cover_specs.ipynb:80]: Do we call it "Metadata Explorer" elsewhere? I've always just called it "Explorer"...

File: data_specs/Fractional_Cover_specs.ipynb [hunk 2]
```diff
@@ -56,7 +56,14 @@
...
+    "\n",
+    "The DE Africa FC service has two components:\n",
+    "\n",
+    "* Fractional Cover, which is estimated from each Landsat scene, providing measurements from individual days.\n",
+    "* Fractional Cover Annual Summary (Percentiles), which provides 10th, 50th, and 90th percentiles estimated independently for the green vegetation, non-green vegetation, and bare soil fractions observed in each calendar year (1st of January - 31st December). \n",
+    "\n",
+    "While the scened based Fractional Cover can be used to study dynamic processes, the annual summaries make it easier to analyse year to year changes. The percentiles provide robust estimates of the low, median and high proportion values observed for each cover type in a year, which can be used to characterise the land cover. Before the percentile calculation, areas of water and cloud cover, as mapped in the Water Observations from Space (WOfS) Feature Layer, are exluded."
```
Comment (alexgleith) [inline data_specs/Fractional_Cover_specs.ipynb:66]:
```suggestion
    "While the scene based Fractional Cover can be used to study dynamic processes, the annual summaries make it easier to analyse year to year changes. The percentiles provide robust estimates of the low, median and high proportion values observed for each cover type in a year, which can be used to characterise the land cover. Before the percentile calculation, areas of water and cloud cover, as mapped in the Water Observations from Space (WOfS) Feature Layer, are exluded."
```

File: data_specs/Fractional_Cover_specs.ipynb [hunk 3]
```diff
@@ -84,16 +94,17 @@
...
-    "|Temporal resolution | 16 days |\n",
-    "|Temporal range| 1984 &ndash; present |\n",
-    "|Parent dataset| Landsat Collection 2 Surface Reflectance |\n",
-    "|Update frequency| Daily |"
+    "|Specification | | |\n",
+    "|----------|-------------|-------------|\n",
+    "|Product name| Fractonal Cover | Fractonal Cover Annual Summary|\n",
+    "|Cell size - X (metres) | 30 | 30 |\n",
+    "|Cell size - Y (metres) | 30 | 30 |\n",
+    "|Coordinate reference system | Universal Transverse Mercator (UTM) | EPSG: 6933 |\n",
```
Comment (alexgleith) [inline data_specs/Fractional_Cover_specs.ipynb:102]:
```suggestion
    "|Coordinate reference system | Cylindrical Equal Area | EPSG: 6933 |\n",
```

**Review by alexgleith — CHANGES_REQUESTED** [review]
A few suggestions.

File: data_specs/Fractional_Cover_specs.ipynb (hunk 1 above)
Comment (fangfy) [reply data_specs/Fractional_Cover_specs.ipynb:80]: I think we used this term before you can download directly from Explorer... we should probably update....

File: data_specs/Fractional_Cover_specs.ipynb (hunk 3 above)
Comment (fangfy) [reply data_specs/Fractional_Cover_specs.ipynb:102]: Are the FC scenes in Cylindrical Equal Area?

Comment (alexgleith) [reply data_specs/Fractional_Cover_specs.ipynb:102]: Oh, nope. Only the summary. Scenes are still UTM.

**Pull request merged**